modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm

```diff
@@ -18,20 +20,9 @@ GLOBAL_LIST_INIT(loadout_helmets, generate_loadout_items(/datum/loadout_item/hea
 	else
 		outfit.head = item_path
 
-
-//readd these on christmas or suffer my wrath
```

Comment:
Any reason these are being removed?

It'd be more annoying for someone to fully readd these around Christmas.

Reply:
They aren't.